### PR TITLE
Use SDK Cloud script to install gcloud

### PR DESCRIPTION
**Pull request by jakule**
Looks like the latest version of Google SDK in Ubuntu repo is broken on ARM64:
```
Errors were encountered while processing:
 google-cloud-sdk
 google-cloud-sdk-firestore-emulator
E: Sub-process /usr/bin/dpkg returned an error code (1)
```
Official Google SDK script seems to work fine.

Alternative is to pin gcloud to v369.00 and wait for Google to release a fix.

### CODE REVIEW

File: build.assets/Dockerfile [hunk 1]
```diff
@@ -22,24 +22,21 @@ ENV LANGUAGE="en_US.UTF-8" \
 ARG BUILDARCH
 
 # Install packages.
-# We install curl first to allow setting up the Google SDK as part of the same layer.
+# Java JRE is resuired by gcloud firestore emulator.
```
Comment (greedy52) [inline build.assets/Dockerfile:25]: typo `resuired`?

File: build.assets/Dockerfile [hunk 2]
```diff
@@ -70,6 +67,11 @@ RUN apt-get update -y --fix-missing && \
     apt-get -y clean && \
     rm -rf /var/lib/apt/lists/*
 
+# Install gcloud SDK and Firestore emulator
+RUN curl -sSL https://sdk.cloud.google.com | bash
+ENV PATH="$PATH:/root/google-cloud-sdk/bin"
+RUN gcloud components install cloud-firestore-emulator beta
```
Comment (greedy52) [inline build.assets/Dockerfile:73]: what's `beta`? do we need it?

Comment (jakule) [reply build.assets/Dockerfile:73]: According to Google docs the firestore emulator is still in beta. gcloud requires to install `beta` package if you want to use it: https://cloud.google.com/sdk/gcloud/reference/beta/emulators/firestore/start

File: build.assets/Dockerfile (hunk 1 above)
Comment (jakule) [reply build.assets/Dockerfile:25]: yes, thanks.

File: build.assets/Dockerfile (hunk 2 above)
Comment (jakule) [reply build.assets/Dockerfile:74]: I added a comment why we need beta package.

**Review by greedy52 — APPROVED** [review]

**Review by atburke — APPROVED** [review]

**Review by smallinsky — APPROVED** [review]

**Comment by greedy52** [comment]
not sure if we can have a daily build for this and push it to our docker repo so we can reference that here. seems a lot of work to rebuild for everyone everywhere

**Pull request merged**